config/application.rb

```diff
@@ -11,6 +11,15 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.1
 
+    # ** Please read carefully **
+    # Change the format of the cache entry.
+    # Changing this to Rails 7.0 means that all new cache entries added to the cache
+    # will have a different format that is not supported by Rails 6.1 applications.
+    # Only remove this value after your application is fully deployed to Rails 7.0
+    # and you have no plans to rollback.
+
+    config.active_support.cache_format_version = 6.1
```

Comment:
Putting this here just to be safe, but it doesn't look like we would have a good reason to need to rollback to 6.1? Could potentially just switch over to the new format

Reply:
We should be safe to move to the new format. We're not explicitly using the active support cache anywhere and the new format is backwards compatible with the 6.1 format so won't invalidate any underlying caches.